content/guides/tutorial-wordpress.md

```diff
@@ -47,7 +47,7 @@ define( 'DB_HOST', getenv("MYSQL_ADDON_HOST").':'.getenv("MYSQL_ADDON_PORT") );
 ```
 
 {{< callout type="warning" >}}
-Excepting MySQL DEV plan, you have to figure the port out in `wp-config.php` with the environment variable `MYSQL_ADDON_HOST` because it is not the default port which is used.
+**Make sure you use the port we give you** in `wp-config.php`! Use the `MYSQL_ADDON_PORT` environment variable for that. We do **not** use the default MySQL port.
```

Comment:
Added clarity to the sentence in order to define who gives the port, otherwise it can be confusing (could sound like clients need to ask the support for the port, or something like that):

```suggestion
**Make sure you use ** in `wp-config.php` the port the add-on dashboard provides. Use the `MYSQL_ADDON_PORT` environment variable for that. We **don't** use the default MySQL port.
```